restraint.spec

```diff
@@ -10,7 +10,7 @@
 %endif
 
 Name:		restraint
-Version:	0.2.2
+Version:	0.2.3
```

Comment:
This is definitely wrong

Reply:
Sorry about that. I need to do this for testing.